Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+﻿
+[color=#ff0000]You must be 16 or older to play. Users under 16 will be banned immediately and be unbanned when they are old enough.[/color]
```

Comment:
I thought it's supposed to be 18 just for legal CYA?

Reply:
nah